functions.sh

```diff
@@ -49,3 +49,15 @@ docker_try_rmi() {
         docker rmi $image_name
     }
 }
+
+sonic_get_version() {
+    local describe=$(git describe --tags)
+    local latest_tag=$(git describe --tags --abbrev=0)
+    BUILD_NUMBER=${BUILD_NUMBER:-dev}
+    ## Cehck if we are on tagged commit
+    if [ "$describe" == "$latest_tag" ]; then
+        echo "$latest_tag"
+    else
+        echo "${latest_tag}.${BUILD_NUMBER}-$(git rev-parse --short HEAD)"
```

Comment:
Latest tag is included into image to have as much info as possible.
Usually if, for instance, feature is supported since SONiC-17.04, then it is true for 17.04.123 (some daily build). That will allow user to know what is available for them.